bin/bench

```diff
@@ -3,16 +3,32 @@
 const path = require('path');
 const program = require('commander');
 const run = require('../lib/run');
+const mkdirp = require('mkdirp').sync;
+const { dirname, readFileSync, writeFileSync } = require('fs');
 
 let input = path.resolve('bench');
 
 program
   .version('0.1.0')
   .arguments('<directory>')
   .option('-i, --iterations <n>', 'The number of iterations to run', parseInt)
-  .action((directory) => input = path.resolve(directory))
-  .parse(process.argv);
+  .action((directory) => {
```

Comment:
The `action` hook only runs when arguments are provided (e.g., `bench some/path`), but it should also be possible to run the command without any arguments (e.g., `bench`), so we should keep the `run()` invocation outside this block.